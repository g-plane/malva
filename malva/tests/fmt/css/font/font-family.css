#container {
  font-family: Inter,
    Roboto,
    "Helvetica Neue",
    "Arial Nova",
    "Nimbus Sans",
    Arial,
    sans-serif;
  font-family:
    Inter,
    Roboto,
    "Helvetica Neue",
    "Arial Nova",
    "Nimbus Sans",
    Arial,
    sans-serif;
}
