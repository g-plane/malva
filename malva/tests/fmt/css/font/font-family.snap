---
source: malva/tests/fmt.rs
---
#container {
  font-family:
    Inter, Roboto, "Helvetica Neue", "Arial Nova", "Nimbus Sans", Arial,
    sans-serif;
  font-family:
    Inter, Roboto, "Helvetica Neue", "Arial Nova", "Nimbus Sans", Arial,
    sans-serif;
}
